hubspot/connection/exc.py

```diff
@@ -17,7 +17,10 @@
 
 class HubspotException(Exception):
     """The base HubSpot error."""
-    pass
+    def __init__(self, message):
+        """ Explicitly makes the message available as the 'message' attribute. """
+        self.message = message
```

Comment:
I see.  I'll play around with removing the overriden str() and also removing my explicit override.